bin/run-tc-ldc93s1.sh

```diff
@@ -17,4 +17,4 @@ python -u DeepSpeech.py \
   --n_hidden 494 --epoch 75 --random_seed 4567 --default_stddev 0.046875 \
   --max_to_keep 1 --checkpoint_dir '/tmp/ckpt' --checkpoint_secs 0 \
   --learning_rate 0.001 --dropout_rate 0.05  --export_dir "/tmp/train" \
-  --use_seq_length False --decoder_library_path "/tmp/ds/libctc_decoder_with_kenlm.so"
+  --use_seq_length=False --decoder_library_path "/tmp/ds/libctc_decoder_with_kenlm.so"
```

Comment:
--nouse_seq_length

Reply:
Nice catch, I'll update all callsites and doc matching `DEFINE_boolean` for consistency.

Reply:
Looks like there is only one callsite, indeed:
```
$ for arg in $(git grep DEFINE_boolean  |cut -d"'" -f2); do echo "${arg}"; git grep -w "\-\-${arg}"; done;
fulltrace
train
test
use_warpctc
remove_export
use_seq_length
bin/run-tc-ldc93s1.sh:  --use_seq_length=False --decoder_library_path "/tmp/ds/libctc_decoder_with_kenlm.so"
native_client/BUILD:        ### Needed by production model produced without "--use_seq_length False"
log_traffic
log_placement
early_stop
alex@portable-alex:~/codaz/Mozilla/DeepSpeech/deepspeech-kdavis$ 
```